File: SEMANA 1/TALLER 1/Ejercicio1.py
import random

valores = [round(random.uniform(1, 100), 2) for _ in range(10)]

suma = sum(valores)
promedio = suma / len(valores)

print(f"Valores generados: {valores}")
print(f"Suma: {suma}")
print(f"Promedio: {promedio}")

File: SEMANA 1/TALLER 1/Ejercicio1.cpp
#include <iostream>
#include <cstdlib>
#include <ctime>
using namespace std;

int main() {
    srand(time(0));

    const int N = 10;
    double valores[N];
    double suma = 0.0;

    for (int i = 0; i < N; i++) {
        valores[i] = (rand() % 10000) / 100.0;
        suma += valores[i];
    }

    double promedio = suma / N;

    cout << "Valores generados: ";
    for (int i = 0; i < N; i++) {
        cout << valores[i] << " ";
    }
    cout << endl;

    cout << "Suma: " << suma << endl;
    cout << "Promedio: " << promedio << endl;

    return 0;
}